IPython/core/debugger.py

```diff
@@ -621,3 +621,13 @@ def do_where(self, arg):
                 self.print_stack_trace()
 
         do_w = do_where
+
+
+def set_trace(frame=None):
```

Comment:
Yes, it should indeed do something. Let me know if you'd prefer to remove the `frame` parameter or add it to `IPython.terminal` as well.